support-frontend/assets/pages/[countryGroupId]/thank-you.tsx

```diff
@@ -82,13 +106,52 @@ export function ThankYou({ geoId }: Props) {
 		return <div>Unable to find contribution type {contributionType}</div>;
 	}
 
-	const isOneOffPayPal =
-		order.paymentMethod === 'PayPal' && order.product === 'Contribution';
-
 	const abParticipations = abTestInit({ countryId, countryGroupId });
 	const showOffer =
 		!!abParticipations.usFreeBookOffer && order.product === 'SupporterPlus';
 
+	const isOneOff = order.product === 'Contribution';
+	const isOneOffPayPal = order.paymentMethod === 'PayPal' && isOneOff;
+	const isSupporterPlus = order.product === 'SupporterPlus';
+	const isNewAccount = order.userTypeFromIdentityResponse === 'new';
+	const csrf = { token: order.csrf };
+
+	const thankYouModuleData = getThankYouModuleData(
+		countryId,
+		countryGroupId,
+		csrf,
+		order.email,
```

Comment:
Instead of storing PPI - could we refactor this to used `isSignedIn`?

I can't see anywhere in the thank you components where we use the email?

Reply:
Think its passed into the <SignInCTA> to construct the SignInUrl on the SignIn thankyou component.